Description: Apache 2 is called "apache2" and not "httpd" under Debian.
Forwarded: not-needed.
--- a/src/vigilo/vigiconf/applications/vigirrd/__init__.py
+++ b/src/vigilo/vigiconf/applications/vigirrd/__init__.py
@@ -17,7 +17,7 @@
     name = "vigirrd"
     priority = -1
     validation = "validate.sh"
-    start_command = "sudo /etc/init.d/httpd reload"
+    start_command = "sudo /etc/init.d/apache2 reload"
     stop_command = None
     generator = generator.VigiRRDGen
     group = "metrology"
--- a/src/vigilo/vigiconf/applications/vigirrd/validate.sh
+++ b/src/vigilo/vigiconf/applications/vigirrd/validate.sh
@@ -19,7 +19,7 @@
 # Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
 ################################################################################
 if [ "$2" == "local" ] ; then
-    ls /etc/init.d/httpd >/dev/null || exit 1
+    ls /etc/init.d/apache2 >/dev/null || exit 1
 fi
 
 export BASE=$1
